qa-kidsfirst.planx-pla.net/portal/gitops.json

```diff
@@ -169,12 +169,6 @@
         }
       ]
     },
-    "arrangerConfig": {
```

Comment:
i don't think keeping the explorer config without providing either an arranger config or a guppy config would work. can you remove the whole data explorer config + link in QA and when you find a config that works, update this PR?